### PR TITLE
Week4 solo branch

### PR DESCRIPTION
**Pull request by mk7run**

### CODE REVIEW

**Comment by mk7run** [comment]
@bontgoy Week4 Solo PR

File: ruby/vampires.rb [hunk 1]
```diff
@@ -0,0 +1,179 @@
+=begin
```
Comment (Bontgoy) [inline ruby/vampires.rb]: Wow great pseudocode!

File: ruby/vampires.rb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    vampire_name = true
+  else
+    vampire_name = false
+  end
+  #Age
+  puts 'What age will you be at the end of the year?'
+    user_age = gets.chomp.to_i #convert input to integer
+  puts 'What is your birth year?'
+    user_birth_year = gets.chomp.to_i #convert input to integer
+  birth_year_to_age = 2017 - user_birth_year
```
Comment (Bontgoy) [inline ruby/vampires.rb]: I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.

File: ruby/vampires.rb [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+  if user_age == birth_year_to_age
+    age_matches = true
+  else
+    age_matches = false
+  end
+  #Garlic Bread
+  good_garlic_answer = false
+  while (not good_garlic_answer) #until good_garlic_answer is true, will loop
```
Comment (Bontgoy) [inline ruby/vampires.rb]: Instead of `while not` try `until`

**Pull request merged**